fluidreview/views.py

```diff
@@ -13,6 +14,10 @@ class WebhookView(APIView):
     permission_classes = (WebhookPermission,)
     authentication_classes = ()
 
+    def handle_exception(self, exc):
+        """Raise any exception with request info instead of returning response with error status/message"""
+        raise FluidReviewException("REST Error (%s). BODY: %s, META: %s" % (exc, self.request.body, self.request.META))
```

Comment:
Instead of putting the exception message in the string, can you chain it like this: `raise FluidReviewException("...") from exc`?

Reply:
Sure, but I'd like to keep `exc.detail` in the string. (`Could not satisfy the request Accept header.`, `You do not have permission to perform this action`).